app/models/program_session.rb

```diff
@@ -1,10 +1,39 @@
 class ProgramSession < ApplicationRecord
-  LIVE = 'live'
-  DRAFT = 'draft'
-  WAITLISTED = 'waitlisted'
+  LIVE = 'live' # confirmed accepted
+  DRAFT = 'draft' # accepted, to be confirmed by organizer
```

Comment:
The comment here would be more like this:
# created by organizer that isn't ready to be published (live)